.pipelines/templates/.builder-release-template.yaml

```diff
@@ -17,6 +17,25 @@ steps:
       vstsFeedPackage: asc-baseline
       vstsPackageVersion: '1.1.0-268'
       downloadDirectory: vhdbuilder/packer
+  - task: DownloadPipelineArtifact@2
+    displayName: 'Download Kata CC UVM artifact'
+    inputs:
+      source: specific
+      project: CloudNativeCompute
+      pipeline: 318279
+      specificBuildWithTriggering: true
+      buildVersionToDownload: 'latest'
+      downloadType: 'single'
+      artifactName: 'drop_container_job_sign_uvm'
+      itemPattern: |
+        **/kata-containers-igvm-debug.img**
+        **/igvm-measurement**
+        **/kata-containers-igvm.img**
+        **/kata-containers.img**
+        **/kata-containers-initrd-base.img**
+        **/reference-info-base64**
+      downloadPath: $(Build.SourcesDirectory)
+    condition: eq(variables.OS_SKU, 'CBLMariner')
```

Comment:
do we also want to do this for OS_SKU=AzureLinux? Henry Li made a change recently to onboard AzureLinux and iirc that also has kata flavor.

Reply:
Good point! That will come with later PRs after preview as we have done multiple rounds of testing with OSSKU=CBLMariner.

OS_SKU=AzureLinux on KataCC needs to be tested more before release.